.env.tests.template

```diff
@@ -0,0 +1,34 @@
+# Mandatory
+MEMBRANE_CORS_ALLOWED_ORIGINS=
+MEMBRANE_FRONTEND=
+MEMBRANE_SECRET_KEY=
+MEMBRANE_CLIENT_PUBLIC_KEYS_DIRECTORY=
+MEMBRANE_SERVER_PRIVATE_KEY=
+MEMBRANE_SERVER_PUBLIC_KEY=
+MEMBRANE_COMM_CONNECTION_STRING=
+MEMBRANE_SENDER_EMAIL=
+
+# Optional
+# MEMBRANE_JWT_ACCESS_TOKEN_EXPIRE_SECONDS=
+# MEMBRANE_JWT_EXPIRE_SECONDS=
+# MEMBRANE_SESSION_LIFETIME_SECONDS=
+# MEMBRANE_SESSION_COOKIE_SECURE=
+# MEMBRANE_SESSION_TYPE=
+# MEMBRANE_TOKEN_BLACKLIST=
+# MEMBRANE_APP_ID_FIELD=
+# MEMBRANE_DATA_FIELD=
+# MEMBRANE_REDIRECT_URL_FIELD=
+# MEMBRANE_EXPIRATION_FIELD=
+# MEMBRANE_ENCODE_ALGORITHM=
+# MEMBRANE_TOKEN_TYPE=
```

Comment:
Not needed

https://datatracker.ietf.org/doc/html/rfc7519#section-5.1

This parameter is
   ignored by JWT implementations; any processing of this parameter is
   performed by the JWT application.  If present, it is RECOMMENDED that
   its value be "JWT" to indicate that this object is a JWT.  While
   media type names are not case sensitive, it is RECOMMENDED that "JWT"
   always be spelled using uppercase characters for compatibility with
   legacy implementations.  Use of this Header Parameter is OPTIONAL.

Reply:
done